Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
       "viewing angles.\nMight cause issues in a small number of games.\n\nIf unsure, select 1x.");
 
   static const char TR_POSTPROCESSING_DESCRIPTION[] = QT_TR_NOOP(
-      "Apply a post-processing effect after finishing a frame.\n\nIf unsure, select (off).");
+      "Applies a post-processing effect after finishing a frame.\n\nIf unsure, select (off).");
 
   static const char TR_SCALED_EFB_COPY_DESCRIPTION[] = QT_TR_NOOP(
-      "Greatly increases quality of textures generated using render-to-texture "
-      "effects.\nRaising the "
-      "internal resolution will improve the effect of this setting.\nSlightly increases GPU "
-      "load and "
-      "causes relatively few graphical issues.\n\nIf unsure, leave this checked.");
+      "Greatly increases the quality of textures generated using render-to-texture "
+      "effects.\n\nSlightly increases GPU load and causes relatively few grpahical issues. "
+      "Raising the internal resolution will improve the effect of this setting. "
+      "\n\nIf unsure, leave this checked.");
   static const char TR_PER_PIXEL_LIGHTING_DESCRIPTION[] = QT_TR_NOOP(
       "Calculates lighting of 3D objects per-pixel rather than per-vertex, smoothing out the "
-      "appearance of lit polygons and making individual triangles less noticeable.\nRarely causes "
-      "slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
+      "appearance of lit polygons and making individual triangles less noticeable.\n\nRarely "
+      "causes slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
   static const char TR_WIDESCREEN_HACK_DESCRIPTION[] = QT_TR_NOOP(
-      "Forces the game to output graphics for any aspect ratio.\nUse with \"Aspect Ratio\" set to "
-      "\"Force 16:9\" to force 4:3-only games to run at 16:9.\nRarely produces good results and "
-      "often partially breaks graphics and game UIs.\nUnnecessary (and detrimental) if using any "
+      "Forces the game to output graphics at any given aspect ratio.\n\nUse with \"Aspect Ratio\" "
+      "set to \"Force 16:9\" to force 4:3-only games to run at 16:9.\n\nRarely produces good results "
+      "and often partially breaks graphics and game UIs.\nUnnecessary (and detrimental) if using any "
       "AR/Gecko-code widescreen patches.\n\nIf unsure, leave this unchecked.");
   static const char TR_REMOVE_FOG_DESCRIPTION[] =
-      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing the overall "
-                 "detail.\nDisabling fog will break some games which rely on proper fog "
-                 "emulation.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing overall "
+                 "detail.\n\nDisabling fog will break some games which rely on proper fog "
+                 "emulation.\n\nIf unsure, leave unchecked.");
   static const char TR_3D_MODE_DESCRIPTION[] = QT_TR_NOOP(
```

Comment:
This description is too many lines as well.
![img](https://qimg.techjargaming.com/i/OPdfBqa5.png)

Reply:
I removed some line breaks, I can remove one more if you need me to.